### ISSUE TITLE
Documentation issues

### ISSUE THREAD

**Issue by grinapo** [fontforge/fontforge#3673]
Some (small) documentation issue. I am reading the docs and trying to take note of the Fresh Eyes Experience.

* There seem to be no doc (or a summary of doc) on actual editing. The first thing I was lost at was that I have been cutting points and there was no doc what is the recommended way to close the spline; what makes the (broken) spline to be extremely tick red line (obstructing the visibility of the points) and how to turn it off; how to find the gap. I missed the basic "what do do when" part for the most obvious cases, like "holes in the spline" (cw/ccw problem), splines carving out parts of other splines (result of join/overlap), the handles getting stuck in "distance only mode" (I *guess* it's due to CTRL key but can be extremely annoying not knowing how to turn it off); general cut-and-merge workflow advices. I do not qualify to write the doc since I'm learning myself. Some of the information is actually present in http://fontforge.github.io/en-US/documentation/interface/editmenu/ and related docs, but extremely hard to find for a beginner.

### Small issues 
* http://fontforge.github.io/en-US/documentation/interface/editmenu/ - merge dead link to http://fontforge.github.io/en-US/documentation/interface/pfaeditmath/#Approximating

**Comment by ctrlcctrlv** [comment]
You've got it all wrong, no offense. The project's documentation is explicitly _not_ for new users. That's what designwithfontforge.com is for. Our documentation assumes a high level of competence already, at the very least having read that tutorial.

**Comment by skef** [comment]
The documentation linked directly from fontforge.github.io is, unfortunately, in a bit of a jumble. The starting point all that detailed either, but in addition to [Design With Fontforge](http://designwithfontforge.com/en-US/index.html) there is [this tutorial](http://fontforge.github.io/editexample.html).

**Comment by grinapo** [comment]
@ctrlcctrlv no offense taken, and I have read through that tutorial. (And, technically, I am not that "beginner" but looking at the documentation through the view of one.) I was looking for all the information missing from that tutorial. If you would like to direct me to the place where I should report problems with that tutorial, I obviously shall move over there.

**Comment by grinapo** [comment]
>  there is [this tutorial](http://fontforge.github.io/editexample.html).

@skef  Ah, you are right that I have not even read the pages with the **Large Scary Red Warning**; I have assumed that they were all worked into the DesignWithFortforge and the reference docs. 

For a newcomer (or someone acting like one *wink* *wink*) the documentation (and its whereabouts) is a bit hard to follow (apart from the really friendly and easy to read DWF (which is sometimes losing its focus and talk more about font design and less about the actual drawing problems)).

I can mention several questions which may arise in the first 30 minutes of enjoying FF which is not quite covered in the docs (any of them, possibly), or if it is then I haven't find them yet. (And yes, I have resolved most of them since I've been playing with unknown programs since the age of C=64 but many doesn't have that lucky adaptive background; also it's great to know what is the _recommended_ way of doing something instead of a possibly over-convoluted solution, but that's just my prefectionist inside.)

And if it wasn't clear I am spending some time with FF and intend to help to create a skeleton or list of wanted doc/tutorial parts, but I'm really not [yet] qualified to teach others, and creating fonts is a sideproject for me; I'm here because FF is an exceptionally great tool and I wish to help the docs to reach that level. [It's a bit offtopic, but the DWF font **design** parts are excellent, what I miss is the detailed **editing** and its Q&A, for really simple things, like the link with the Large Scary Warning.]

**Comment by ctrlcctrlv** [comment]
No not at all. The Fontforge documentation that we keep up to date is not even on that website anymore. We don't know how to update it I don't think. Or if one of us does that person isn't doing it. 

The best documentation is in the doc/html folder. Hopefully your distribution is installing that for you. I also maintain a copy of the George Williams' documentation at my website pfaedit.org, which was George's website before he moved to SourceForge and the domain lapsed.

**Comment by ctrlcctrlv** [comment]
Also now that I'm reading pdaedit.org again I think I need to write another history page. @davelab6's involvement is pretty low these days but he does show up occasionally. So much has happened since I wrote that in...2015 I think?

Anyway @davelab6 is the only one who can remove the "scary warning", and I've always thought he should do so. Let's see if my mentioning him brings clarity :)

**Comment by skef** [comment]
@jtanx recently re-synced the html docs "on" (but not really linked from) the website with #3530. `[doc/synchweb.py](https://github.com/fontforge/fontforge/blob/master/doc/synch_web.py)` is what adds that message, so it could be modified there and re-run. None of this affects the Kramdown (a Markdown variant) version of the documents, although I did some work on those by hand with https://github.com/fontforge/fontforge.github.io/pull/127, making all of them except the python document current.

**Comment by ctrlcctrlv** [comment]
@skef I didn't know that fontforge.org was hosted by Github, thanks

**Comment by grinapo** [comment]
May I make a suggestion of writing a new page describing briefly 
- which documentation is "official" and "actual";
- which is not incorporated elsewere and still shall be read but may contain outdated information;
- which documentation is already incorporated and shall not be used at all;
- which documentation is, though not from the developers, shall be used
as a part of getting-started? 
As summarizing where to get information, where to see the FAQ, where to get help, where to report issues (about documentation and the code; these are partly covered already in various places).

The Scary Warning may specifically refer to that so there would be information for the user to decide what to do.

**Comment by ctrlcctrlv** [comment]
> (Is "pdaedit.org" in reality http://pfaedit.org/ you were referring, @ctrlcctrlv ?)

Yes

**Comment by davelab6** [comment]
I'm not the only one who can remove the warning, anyone can make a PR to do
it and if it gets merged then it's happened.

I'm happy to make time for a video call if someone wants to talk to me
about the history.

http://metapolator.com/home/#project-history and
https://typedrawers.com/discussion/comment/15260/#Comment_15260 are pretty
good write ups from a couple years ago when I was pushing more on the libre
tools side of my libre fonts triangle; the other two sides are knowledge
about how to do type design using those tools (www.craftingtype.com and
www.designwithfontforge.com and www.oert.org so far) and businesses to pay
people to put the knowledge and tools into action to actually make fonts -
which is my main focus (fonts.google.com)

**Comment by davelab6** [comment]
https://github.com/davelab6/matd-dissertation/blob/gh-pages/dave_crossland_matd2008_dissertation.pdf
might also be helpful

**Comment by davelab6** [comment]
I would be willing to do a public video call next week. NYC business hours.

@eliheuer are you up for it? :)
